/kuka_lbr/wbc/joints:
  ros__parameters:
    noise_std_dev: 1e-4
    initial_joint_state:
        name: ["kuka_lbr_r_joint_1","kuka_lbr_r_joint_2","kuka_lbr_r_joint_3","kuka_lbr_r_joint_4","kuka_lbr_r_joint_5","kuka_lbr_r_joint_6","kuka_lbr_r_joint_7",
               "kuka_lbr_l_joint_1","kuka_lbr_l_joint_2","kuka_lbr_l_joint_3","kuka_lbr_l_joint_4","kuka_lbr_l_joint_5","kuka_lbr_l_joint_6","kuka_lbr_l_joint_7"]
        position: [0.0,1.0,0.0,1.5,0.0,-1.0,0.0,
                   0.0,1.0,0.0,1.5,0.0,-1.0,0.0]
        velocity: [0.0,0.0,0.0,0.0,0.0,0.0,0.0,
                   0.0,0.0,0.0,0.0,0.0,0.0,0.0]
        effort: [0.0,0.0,0.0,0.0,0.0,0.0,0.0,
                 0.0,0.0,0.0,0.0,0.0,0.0,0.0]
